core/js/directives/StopEventDirective.js

```diff
@@ -0,0 +1,18 @@
+angular.module('ev-fdm')
+    .directive('evStopEvent', function () {
```

Comment:
stop is not really explicit. It also could mean stopImmediatePropagation or maybe preventDefault ?
ev-stop-propagation might be more accurate ? (and more reflected to the dom api)
What do you think ?